scanner/metadata/taglib.go

```diff
@@ -1,83 +1,13 @@
+//+build !cgo
+
 package metadata
 
 import (
-	"errors"
-	"os"
-
-	"github.com/dhowden/tag"
-	"github.com/navidrome/navidrome/log"
-	"github.com/navidrome/navidrome/scanner/metadata/taglib"
+	"fmt"
 )
 
-type taglibMetadata struct {
-	baseMetadata
-	hasPicture bool
-}
-
-func (m *taglibMetadata) Title() string  { return m.getTag("title", "titlesort", "_track") }
-func (m *taglibMetadata) Album() string  { return m.getTag("album", "albumsort", "_album") }
-func (m *taglibMetadata) Artist() string { return m.getTag("artist", "artistsort", "_artist") }
-func (m *taglibMetadata) Genre() string  { return m.getTag("genre", "_genre") }
-func (m *taglibMetadata) Year() int      { return m.parseYear("date", "_year") }
-func (m *taglibMetadata) TrackNumber() (int, int) {
-	return m.parseTuple("track", "tracknumber", "_track")
-}
-func (m *taglibMetadata) Duration() float32 { return m.parseFloat("length") }
-func (m *taglibMetadata) BitRate() int      { return m.parseInt("bitrate") }
-func (m *taglibMetadata) HasPicture() bool  { return m.hasPicture }
-
 type taglibExtractor struct{}
 
 func (e *taglibExtractor) Extract(paths ...string) (map[string]Metadata, error) {
-	mds := map[string]Metadata{}
```

Comment:
Adding a mandatory dependency on cgo isn't super-duper great :/

Reply:
I didnt add that o.O I just added a build constrain since there wasnt one before. Also since this uses sqlite it depends on CGO anyway

Reply:
Yes, that's why I didn't bother to implement this when I added taglib. But it is a nice to have anyway, thanks!

Reply:
Oh, my bad.

Yeah, the sqlite thingy is a bit annoying :/